app/uk/gov/hmrc/exports/models/declaration/submissions/Action.scala

```diff
@@ -20,11 +20,7 @@ import java.time.LocalDateTime
 
 import play.api.libs.json.Json
 
-case class Action(
-  requestType: RequestType,
-  conversationId: String,
-  requestTimestamp: LocalDateTime = LocalDateTime.now()
-)
+case class Action(id: String, requestType: RequestType, requestTimestamp: LocalDateTime = LocalDateTime.now())
```

Comment:
Can we add scaladoc which explains that `id` here is `converstationId` from `customs-notifications` in most cases - this is something which is handy express in code